src/libs/actions/PersonalDetails.ts

```diff
@@ -41,20 +41,19 @@ Onyx.connect({
 });
 
 /**
- * Returns the displayName for a user
+ * Creates a new displayName for a user based on passed personal details or login.
  */
-function getDisplayName(login: string, personalDetail: Pick<PersonalDetails, 'firstName' | 'lastName'> | null): string {
+function createDisplayName(login: string, personalDetails: Pick<PersonalDetails, 'firstName' | 'lastName'>): string {
     // If we have a number like +15857527441@expensify.sms then let's remove @expensify.sms and format it
     // so that the option looks cleaner in our UI.
     const userLogin = LocalePhoneNumber.formatPhoneNumber(login);
-    const userDetails = personalDetail ?? allPersonalDetails?.[login];
```

Comment:
```diff
- ?? allPersonalDetails?.[login];
```
For 2nd case, this is major change. Others are just code refactor, not affecting logic, right?

Reply:
logic change I mean. Others are just variable re-naming, etc